GoogleDataTransport/GDTCORTests/Unit/GDTCORClockTest.m

```diff
@@ -81,4 +81,36 @@ - (void)testIsAfter {
   XCTAssertFalse([clock2 isAfter:clock1]);
 }
 
+- (void)testUptime {
+  NSTimeInterval timeDiff = 1;
+  GDTCORClock *clock1 = [GDTCORClock snapshot];
+  [[NSRunLoop currentRunLoop] runUntilDate:[NSDate dateWithTimeIntervalSinceNow:timeDiff]];
+  GDTCORClock *clock2 = [GDTCORClock snapshot];
+
+  XCTAssertGreaterThan(clock2.uptimeNanoseconds, clock1.uptimeNanoseconds);
+  NSTimeInterval uptimeDiff =
+      (clock2.uptimeNanoseconds - clock1.uptimeNanoseconds) / (NSTimeInterval)NSEC_PER_SEC;
```

Comment:
I would replace NSTimeInterval with "double" as as it is weird to convert NSEC_PER_SEC to NSTimeInterval.